File: DayTwo/Java/BookSuggestionSystem.java
import java.util.Scanner;

import java.util.Random;

import java.util.ArrayList;

import java.util.Arrays;

public class BookSuggestionSystem{

    public static String suggestBook (ArrayList<String> books){

    Random rand = new Random();
    
    int index = rand.nextInt(books.size());

    String randomBook = books.get(index);

    return randomBook;}

    public static int getRandomPage(){

    Random rand = new Random();

    int randomPage = rand.nextInt(100) + 1;

    return randomPage;}

    public static ArrayList<String> addBook (ArrayList<String> books, String newBook){

    if (books.contains(newBook)) return books;

    else books.add(newBook);

    return books;}

    public static ArrayList<String> removeBook(ArrayList<String> books, String oldBook){

    books.remove(oldBook);

    return books;}

    public static ArrayList<String> updateBook(ArrayList<String> books, String oldTitle, String newTitle){
    
    int index = books.indexOf(oldTitle);

    books.set(index, newTitle);

    return books;
    }

    public static String showAllBooks (ArrayList<String> books){

    int bookNumber = 1;

    String display = "";

    for (String element : books){

    display += String.format("%d. %s%n", bookNumber++,element);
    } 

    return display;  
    }

}

File: DayTwo/Python/book_suggestion_system.py
import random

#def return_book_library():
#
#    book_library = ["My Journey in Semicolon", "Who Moved My Cheese", "Atomic Habit", "Subtle Art of Not Giving a Fuck"]
#
#    return book_library

def suggest_book(book_library):

    random_book = random.choice(book_library)

    return random_book

def get_random_page():

    random_page = random.randint(1,100)

    return random_page

def add_book(book_library,new_book):

    book_library.append(new_book)

    return book_library

def remove_book(book_library, old_book):

    book_library.remove(old_book)

    return book_library

def update_book(book_library,old_book,new_book): 
   
    index = book_library.index(old_book)

    book_library[index] = new_book

    return book_library

def show_all_books(book_library):    

    book_number = 1

    for book in book_library:

        print(f"{book_number}. {book}")

        book_number = book_number + 1

    print()

        

    
    
